applications/luci-app-attendedsysupgrade/root/usr/share/luci/menu.d/luci-app-attendedsysupgrade.json

```diff
@@ -0,0 +1,31 @@
+{
+	"admin/system/attendedsysupgrade": {
+		"title": "Attended Sysupgrade",
+		"order": 60,
+		"action": {
+			"type": "firstchild"
+		},
+		"depends": {
+			"acl": [ "luci-app-attendedsysupgrade" ],
```

Comment:
Should be `"acl": [ "attendedsysupgrade" ]`

Reply:
Instead I renamed the acl.d file to luci-app-attendedsysupgrade to follow the style from e.g. luci-app-firewall.